miranda/convert/data/ets_grnch_attrs.json

```diff
@@ -0,0 +1,90 @@
+{
+  "Header": {
+    "Conventions": "CF-1.9",
+    "_miranda_version": true,
+    "contact": "Dr. Annie Poulin (annie.poulin@etsmtl.ca)",
+    "creation_date": "2022-07-16T11:22:42Z",
+    "institution": "ETS",
+    "institution_name": "École de Technologie Supérieure (ETS), Hydrology, Climate and Climate Change Laboratory (HC3)",
+    "license": "Creative Commons Attribution 4.0 International (CC BY 4.0)",
+    "license_type": "OSI approved",
+    "license_url": "https://creativecommons.org/licenses/by/4.0",
+    "processing_level": "raw",
+    "project": "Info-CRUE",
+    "realm": "atmos",
+    "source": "GRNCH_QC",
+    "table_date": "2022-12-14",
+    "table_id": "ets_grnch",
+    "title": "Gridded North American Climate from the HC3 Laboratory (Québec Version)",
+    "type": "reconstruction",
+    "version": "v1.2"
+  },
+  "variable_entry": {
+    "Latitude": {
+      "_cf_variable_name": "lat",
+      "axis:": "Y",
+      "long_name": "Latitude",
+      "standard_name": "latitude",
+      "units": "degrees_north"
+    },
+    "Longitude": {
+      "_cf_variable_name": "lon",
+      "axis:": "X",
+      "long_name": "Longitude",
+      "standard_name": "longitude",
+      "units": "degrees_east"
+    },
+    "P": {
+      "_cf_variable_name": "pr",
+      "_corrected_units": "mm d-1",
+      "_transformation": "amount2rate",
+      "cell_measures": "area: areacella",
+      "cell_methods": "area: mean time: mean",
+      "history": "",
+      "long_name": "precipitation",
+      "standard_name": "precipitation_flux",
+      "units": "kg m-2 s-1"
+    },
+    "T": {
+      "_cf_variable_name": "tas",
+      "_corrected_units": "degC",
+      "_transformation": "amount2rate",
+      "cell_measures": "area: areacella",
+      "cell_methods": "area: mean time: mean",
+      "history": "",
+      "long_name": "Daily Mean Near-Surface Air Temperature",
+      "standard_name": "air_temperature",
+      "units": "K"
+    },
+    "Time": {
+      "_cf_variable_name": "time",
+      "axis": "T",
+      "calendar": "standard",
+      "long_name": "time",
+      "standard_name": "time",
+      "units": "days since 1970-01-01"
```

Comment:
As said above, you don't need to set the `units` and `calendar` if the data has a datetime dtype.